### PR TITLE
feat: warn about sdk update with feature dependencies

### PR DESCRIPTION
**Pull request by kwasniew**
## About the changes

Screenshot 2023-10-17 at 11 34 52

### CODE REVIEW

File: frontend/src/component/feature/Dependencies/DependenciesUpgradeAlert.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+};
+
+const DependenciesDocsLink = () => {
+    return (
+        <a
+            href='https://docs.getunleash.io/reference/dependent-features#client-sdk-support'
+            target='_blank'
+            rel='noreferrer'
+        >
+            Read more
```
Comment (thomasheartman) [inline frontend/src/component/feature/Dependencies/DependenciesUpgradeAlert.tsx:19]: This link text doesn't say anything about what the link points to. It should be possible to understand what a link points to without any context. Can we do something like this instead?
```suggestion
            Client SDK support for feature dependencies
```

Potentially preface the link (outside of the `a` tag) with "Read more about"?

File: frontend/src/component/feature/Dependencies/AddDependencyDialogue.tsx [hunk 2]
```diff
@@ -189,6 +190,8 @@ export const AddDependencyDialogue = ({
             secondaryButtonText='Cancel'
         >
             <Box>
+                <DependenciesUpgradeAlert/>
+                <br />
```
Comment (thomasheartman) [inline frontend/src/component/feature/Dependencies/AddDependencyDialogue.tsx:194]: Why do we use the `br` tag? Shouldn't spacing between elements be handled by styling?

**Review by thomasheartman — APPROVED** [review]
It's probably fine to put this in, but I have a few comments. The important one is around accessibility. The other one is more a nit, but I see we've already done that.

File: frontend/src/component/feature/Dependencies/DependenciesUpgradeAlert.tsx (hunk 1 above)
Comment (kwasniew) [reply frontend/src/component/feature/Dependencies/DependenciesUpgradeAlert.tsx:19]: fixing

**Pull request merged**